halo/healthcheck.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
not sure `scratch` has `sh` installed?

Reply:
Yep it definitely doesn't, but I am not even there yet: I'm struggling to even copy the scripts into the right contianer.